packages/passport/src/index.ts

```diff
@@ -0,0 +1,298 @@
+import { createEIP1193Provider, type WalletInit } from '@web3-onboard/common'
```

Comment:
```suggestion
import type { WalletInit } from '@web3-onboard/common'
```
Can you please split this out as well and dynamically import `createEIP1193Provider` within the `getInterface` method?

Reply:
done